src/njoy21/input/LEAPR/Card11.hpp

```diff
@@ -0,0 +1,20 @@
+class Card11 {
+public:
+  #include "njoy21/input/LEAPR/Card11/Delta.hpp"
+  #include "njoy21/input/LEAPR/Card11/Ni.hpp"
+
+  Argument < Delta > delta;
+  Argument < Ni    > ni;
```

Comment:
That was by far my favorite input to write. Can we try and make more monte python references? I mean the naming is silly enough